src/tools/annotation/AngleTool.js

```diff
@@ -275,10 +275,9 @@ export default class AngleTool extends BaseAnnotationTool {
       const str = '00B0'; // Degrees symbol
 
       return (
-        toGermanNumberStringTemp(data.rAngle) +
+        localization.localizeNumber(data.rAngle) +
         String.fromCharCode(parseInt(str, 16)) +
         suffix
```

Comment:
Should we localize the suffix of the angle tool?

Reply:
A question for @LisaFrauenstein maybe?